task/eaas-provision-space/0.1/eaas-provision-space.yaml

```diff
@@ -58,27 +60,26 @@ spec:
             ;;
         esac
 
-        cat <<EOF > space_request.yaml
-        apiVersion: toolchain.dev.openshift.com/v1alpha1
-        kind: SpaceRequest
+        cat <<EOF > namespace_claim.yaml
+        apiVersion: eaas.konflux-ci.dev/v1alpha1
+        kind: Namespace
         metadata:
-          generateName: eaas-spacerequest-
+          name: $SECRET
           namespace: $NAMESPACE
           ownerReferences:
           - apiVersion: tekton.dev/v1
             kind: $OWNER_KIND
             name: $OWNER_NAME
             uid: $OWNER_UID
         spec:
-          tierName: $TIER_NAME
-          targetClusterRoles:
-          - cluster-role.toolchain.dev.openshift.com/eaas
+          writeConnectionSecretToRef:
+            name: $SECRET-secret
         EOF
 
-        NAME=$(oc create -f space_request.yaml -o=jsonpath='{.metadata.name}')
+        NAME=$(oc create -f namespace_claim.yaml -o=jsonpath='{.metadata.name}')
 
-        if oc wait spacerequests $NAME --for=condition=Ready --timeout=5m; then
-          secretRef=$(oc get spacerequests $NAME -o=jsonpath='{.status.namespaceAccess[0].secretRef}')
+        if oc wait namespaces.eaas.konflux-ci.dev "$NAME" --for=condition=Ready --timeout=5m; then
+          secretRef=$SECRET-secret
```

Comment:
This variable isn't necessary. It'd be better to use the `$SECRET` env var directly anyway which should include this `-secret` postfix.